### PR TITLE
feat(Docs): New release notes: docs v4-5-2024

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* Cross application tracing is deprecated and won't be supported in future APM agent versions. We updated the docs to reflect this.
+* Removed docs related to OpenTelemetry cumulative metrics, which was deprecated in 2023.
+
+### Minor changes
+
+* Updated API docs links related to Node.js [AI monitoring customization](/docs/ai-monitoring/customize-agent-ai-monitoring).
+* Clarified which [Data Plus features](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-prices) apply to Standard, Pro, and Enterprise accounts.
+* Clarified that when you [update the Java APM agent](/docs/apm/agents/java-agent/installation/update-java-agent) and are using one of our [Java agent APIs](/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api), you need to update the API dependencies as well.
+* Updated Python APM agent version support to 9.8.0 (from 9.9.0) to monitor [AI libraries and frameworks](/docs/apm/agents/python-agent/getting-started/compatibility-requirements-python-agent/#digital-intelligence-platform).
+* Deleted errors inbox legacy EOL announcements that are no longer relevant.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx:28]: I think we can lose this one

File: src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* Updated link to ServiceNow in [destination notification](/docs/alerts-applied-intelligence/notifications/notification-integrations) instructions.
+* Our infrastructure agent supports Elastic container service collects all [EC2 node container metrics from the Docker API](/docs/infrastructure/install-infrastructure-agent/linux-installation/container-instrumentation-infrastructure-monitoring) with Cgroup version V2.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx:30]: Some kind of grammatical issue here I think

File: src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Updated information about how to [find OpenTelemetry `singlestore` data](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-singlestore/#view-data) in the UI.
+* Updated CodeStream docs related to [log search](/docs/codestream/observability/log-search) and [performance monitoring](/docs/codestream/observability/performance-monitoring).
+* Added a link to the [Java agent user attributes](/docs/errors-inbox/error-users-impacted) supported by errors inbox.
+* [NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/queries-nrql_screenshot-full_nrql-alert-conditions/#condition-tips) evaluate data aggregated in windows from 30 seconds to six hours.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx:34]: Struggling to parse this one

**Review by austin-schaefer — APPROVED** [review]

File: src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-4-5-2024.mdx:30]: Good catch. A tricky technology-based grammatical tangle :)

**Comment by paperclypse** [comment]
netlify build

**Pull request merged**